### PR TITLE
docs(readme): add webpack config example

### PR DESCRIPTION
**Pull request by budmc29**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

Add simple `webpack` config example, as suggested here: https://github.com/webpack-contrib/webpack-serve/issues/112#issuecomment-386369962
This will make `webpack-serve` more user friendly to beginners.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,25 @@ however, you can utilize any of the options above _in tandem_ with
 can be useful for setups with multiple configs that share common options for
 `webpack-serve`, but require subtle differences.
 
+### `webpack.config.js` example
```
Comment (michael-ciniawsky) [inline README.md]: example => Example

Comment (michael-ciniawsky) [reply README.md]: `'\n'` after the header

File: README.md [hunk 2]
```diff
@@ -126,6 +126,25 @@ however, you can utilize any of the options above _in tandem_ with
...
+```js
+  const path = require('path');
+
+  const { NamedModulesPlugin } = require('webpack');
```
Comment (michael-ciniawsky) [inline README.md]: Please remove this as this plugin isn't needed anymore since `webpack >= v4.0.0`, which will cause confusion for new users :)

File: README.md (hunk 1 above)
Comment (budmc29) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (budmc29) [reply README.md]: Fixed, and good tip, I was unaware of that.

File: README.md [hunk 3]
```diff
@@ -140,8 +139,6 @@ can be useful for setups with multiple configs that share common options for
       filename: './output.js',
       path: path.resolve(__dirname),
     },
-    plugins: [new NamedModulesPlugin()],
-    serve: {},
```
Comment (michael-ciniawsky) [inline README.md]: Was this removal intentional ? :)

Comment (budmc29) [reply README.md]: Yes, to reduce the example, but now that you mention it, it's good to have it even if it's empty.

File: README.md [hunk 4]
```diff
@@ -126,6 +126,23 @@ however, you can utilize any of the options above _in tandem_ with
 can be useful for setups with multiple configs that share common options for
 `webpack-serve`, but require subtle differences.
 
+### `webpack.config.js` Example
+
+```js
+  const path = require('path');
```
[the remark this answers is not in the document]
Comment (budmc29) [reply README.md]: Is it acceptable to fix a few other indentation in the readme file, or that's not accepted as part of this PR?

[the remark this answers is not in the document]
Comment (michael-ciniawsky) [reply README.md]: If you like to do it please go for it (appreciated)

[the remark this answers is not in the document]
Comment (budmc29) [reply README.md]: Fixed

**Comment by budmc29** [comment]
Thank you as well.
